packages/nextcloud/packages/nextcloud_test_presets/docker/presets/server/28.0

```diff
@@ -3,8 +3,8 @@ COOKBOOK_URL=https://github.com/christianlupus-nextcloud/cookbook-releases/relea
 COOKBOOK_CHECKSUM=sha256:ba96d3773bb5ec11fab5e860094713c41bd354e3b7d4aae2ff7def19de149f2c
 DROP_ACCOUNT_URL=https://packages.framasoft.org/projects/nextcloud-apps/drop-account/drop_account-2.6.1.tar.gz
 DROP_ACCOUNT_CHECKSUM=sha256:226e84712aa61d62e99eaa834abc7cd4496ed07289830af06cefeb58c4135123
-NEWS_URL=https://github.com/nextcloud/news/releases/download/25.0.0-alpha9/news.tar.gz
-NEWS_CHECKSUM=sha256:8c7d45fc8648b65644a6c7e05e14688e2603e9911586cb91bf3f10b346775052
+NEWS_URL=https://github.com/nextcloud/news/releases/download/25.0.0-alpha10/news.tar.gz
+NEWS_CHECKSUM=sha256:74268ce7ec95bbf93edd2334e3b597d48ddbffa8429b3cad68f9d660a46f381d
```

Comment:
This version should not have been bumped because alpha10 dropped 28 support. We can still use alpha9 for 28 instead.

Reply:
So this seems to be a limitation of the Appstore as the releases are grouped and the alpha1-9 releases are not listed anymore.